Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8_9_Object Classification_(A).md

```diff
@@ -0,0 +1,17 @@
+## Title : Name of the Topic
+🔴 **Type of Content** : Audio
+
+🔴 **Domain** : Machine Learning
+
+🔴 **Module** : ML_For_Sequential_Data_&_Visual_Data
+
+
+*********************************************************************
+
+### Click here for the audio file,
+
+
+[![Object Detection](https://github.com/snoice45/winter-of-contributing/blob/Machine_Learning/Machine_Learning/ML_For_Sequential_Data_%26_Visual_Data/Assets/ml.jpg)](https://drive.google.com/file/d/1aXNnsMIhLVdT-tonqTdMZBA2WPmMMVRn/view?usp=sharing)
```

Comment:
`![[Object Detection](https://github.com/snoice45/winter-of-contributing/blob/Machine_Learning/Machine_Learning/ML_For_Sequential_Data_%26_Visual_Data/Assets/ml.jpg)](https://drive.google.com/file/d/1aXNnsMIhLVdT-tonqTdMZBA2WPmMMVRn/view?usp=sharing)`

correct syntax
@snoice45